Jenkinsfile

```diff
@@ -80,6 +80,24 @@ pipeline {
             '''
           }
         }
+        stage('Nodejs client tests') {
+          agent { label 'medium' }
+          steps {
+            checkout scm
+            sh '''
+              export HOME=$(pwd)
+              export LANG=en_US.UTF-8
+              test -d env && rm -rf env
+              python3.8 -m venv env
+              . env/bin/activate
+              python3.8 -m pip install -U cr8
```

Comment:
`python3.8` here probably doesn't refer to `env/bin/python`, the `venv` module afaik only creates `python` and `python3` symlinks.